adal/src/main/java/com/microsoft/aad/adal/AcquireTokenWithBrokerRequest.java

```diff
@@ -64,6 +64,12 @@ AuthenticationResult acquireTokenWithBrokerSilent()
                 .isNullOrBlank(mAuthRequest.getUserId())) {
             Logger.v(TAG, "User is specified for background(silent) token request, trying to acquire token silently.");
             authenticationResult = mBrokerProxy.getAuthTokenInBackground(mAuthRequest, brokerEvent);
+            if (null != authenticationResult) {
```

Comment:
If the AuthenticationException (which may contain the server error from broker in some cases) is thrown during the getAuthTokenInBackground calling, will this block still be reached?

Reply:
See commits below - the confusion here should now be resolved as this path is only taken for cached tokens, which will no longer track these fields.

See commit 6361c06d9a92d54c18b6381b5456099c5ce69c1e -- removes conditional setting of `server_error`, `server_suberror`, and `rt_age`

See commit [`a8403be`](https://github.com/AzureAD/azure-activedirectory-library-for-android-unity-fork/pull/132/commits/a8403be25e874ca54fe77836a52436b25a63b316) -- removes fields from `TokenCacheEvent`